src/views/home/Footer.tsx

```diff
@@ -55,11 +55,11 @@ const Footer: VFC = () => {
               <li>
                 <InternalLink href="/nfts/new">Mint NFTs</InternalLink>
               </li>
-              <li>
+              {/* <li>
```

Comment:
intentional?

Reply:
Yeah this isn't active yet so good to hide.